users/vieting/experiments/switchboard/ctc/feat/experiments.py

```diff
@@ -776,6 +784,144 @@ def run_scf_baseline():
+
+    (
+        returnn_datasets,
+        rasr_loss_corpus_path,
+        rasr_loss_corpus_segments,
+        rasr_loss_lexicon_path,
+        dev_corpora,
+    ) = get_datasets(use_multi_proc_dataset=True, pre_process=CodeWrapper("audio_perturb_runner.run"))
+    returnn_args = {
+        "batch_size": 5000,
+        "rasr_binary_path": RASR_BINARY_PATH,
+        "rasr_loss_corpus_path": rasr_loss_corpus_path,
+        "rasr_loss_corpus_segments": rasr_loss_corpus_segments,
+        "rasr_loss_lexicon_path": rasr_loss_lexicon_path,
+        "datasets": returnn_datasets,
+        "conformer_type": "wei",
+        "specaug_old": {"max_feature": 15},
+        "audio_perturbation": True,
+        "use_multi_proc_dataset": True,
+    }
+    feature_args = {"class": "ScfNetwork", "size_tf": 256 // 2, "stride_tf": 10 // 2}
+    lr_args = {
+        "peak_lr": 4e-4,
+        "start_lr": 1.325e-05,
+        "end_lr": 1e-5,
+        "increase_epochs": 180,
+        "decrease_epochs": 180,
+        "final_epochs": 0,
+    }
+
+    perturbation_args = [
+        {"speed": {"prob": 0.6, "minimum": 0.8, "maximum": 1.2}},
+        {"speed": {"prob": 0.6, "minimum": 0.7, "maximum": 1.3}},
+        {"speed": {"prob": 0.5, "minimum": 0.9, "maximum": 1.1}},
+        {"speed": {"prob": 0.5, "minimum": 0.8, "maximum": 1.2}},
+        {"speed": {"prob": 0.5, "minimum": 0.7, "maximum": 1.3}},
+        {"speed": {"prob": 0.4, "minimum": 0.9, "maximum": 1.1}},
+        {"speed": {"prob": 0.4, "minimum": 0.8, "maximum": 1.2}},
+        {"speed": {"prob": 0.4, "minimum": 0.7, "maximum": 1.3}},
+        {"tempo": {"prob": 0.4, "minimum": 0.9, "maximum": 1.1}},
+        {"tempo": {"prob": 0.4, "minimum": 0.8, "maximum": 1.2}},
+        {"tempo": {"prob": 0.4, "minimum": 0.7, "maximum": 1.3}},
+        {"tempo": {"prob": 0.5, "minimum": 0.9, "maximum": 1.1}},
+        {"tempo": {"prob": 0.5, "minimum": 0.8, "maximum": 1.2}},
+        {"tempo": {"prob": 0.5, "minimum": 0.7, "maximum": 1.3}},
+        {"tempo": {"prob": 0.6, "minimum": 0.9, "maximum": 1.1}},
+        {"tempo": {"prob": 0.6, "minimum": 0.8, "maximum": 1.2}},
+        {"tempo": {"prob": 0.6, "minimum": 0.7, "maximum": 1.3}},
+        {"preemphasis": {"prob": 0.9, "minimum": 0.9, "maximum": 1.0}},
+        {"preemphasis": {"prob": 0.9, "minimum": 0.8, "maximum": 1.0}},
+        {"preemphasis": {"prob": 0.8, "minimum": 0.9, "maximum": 1.0}},
+        {"preemphasis": {"prob": 0.8, "minimum": 0.8, "maximum": 1.0}},
+        {"codecs": [{"encoding": "ULAW", "prob": 0.4}]},
+        {"codecs": [{"encoding": "ULAW", "prob": 0.6}]},
+        {"non_linearity": {"prob": 0.4, "minimum": 0.1, "maximum": 0.3}},
+        {"non_linearity": {"prob": 0.4, "minimum": 0.2, "maximum": 0.4}},
+        {"non_linearity": {"prob": 0.6, "minimum": 0.1, "maximum": 0.3}},
+        {"non_linearity": {"prob": 0.6, "minimum": 0.2, "maximum": 0.4}},
+    ]
+
+    def process_args(args):
```

Comment:
```suggestion
    def process_args(args: Dict[str, Any]):
```
Plus `from typing import Any, Dict` at the top.